Jenkinsfile

```diff
@@ -43,5 +43,4 @@
 
 packageBuildingPipelineDAOSTest(['distros': ['el8', 'centos7', 'leap15', 'ubuntu20.04'],
                                  'make args': 'DISTRO_VERSION_EL8=8.5',
-                                 'rpmlint_rpms_allow_errors': true,
```

Comment:
Thanks for catching this.  We generally never want to use this option.  It's a huge hammer for very particular cases.